tasks/snapshot.yml

```diff
@@ -15,39 +15,41 @@ vars:
 
 tasks:
   directory:
-    desc: "Creates a snapshot of the current working directory e.g. task snapshot:directory -- /tmp/release"
+    desc: "Creates a snapshot of the current working directory"
     summary: |
       Creates a snapshot of the current working directory
 
       .git, .gitignore, files listed in .drainpipeignore, and .drainpipeignore
       itself are not added. .drainpipeignore uses the same format as gitignore.
 
-      The target directory should be provided as an argument:
-      task snapshot:directory -- /tmp/release
+        usage: task snapshot:directory o=/tmp/release
+
+        o=<file>   Write the archive to <file>
```

Comment:
This is a directory, not a file?

I think renaming `o` could be clearer. How about `output_directory`?